rockcraft.yaml

```diff
@@ -9,16 +9,17 @@ description: |
 
 version: "v2.1.1_22.04_1"
 license: Apache-2.0
-base: ubuntu:22.04
+build-base: ubuntu:22.04
+base: bare
 platforms:
   amd64:
+run-user: _daemon_
 
 services:
   mlflow-server:
     override: replace
     command: /usr/bin/python3
     startup: enabled
-    user: ubuntu
 
 parts:
   security-team-requirement:
```

Comment:
Could you replace `security-team-requirement` with the following. It is proper way to get all packages:
```
    security-team-requirement:
      plugin: nil
      override-build: |
        mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
        (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
         dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
         > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
```